subsys/bluetooth/mesh/Kconfig

```diff
@@ -1501,8 +1501,7 @@ choice BT_MESH_CRYPTO_LIB
 
 config BT_MESH_USES_MBEDTLS_PSA
 	bool "mbed TLS PSA"
-	select MBEDTLS
-	select MBEDTLS_PSA_CRYPTO_C
+	select PSA_CRYPTO
```

Comment:
I'm not sure if this is correct change, since crypto library is not selected via `PSA_CRYPTO_PROVIDER` choice in Mesh.

With this change, it is possible to choose both `BT_MESH_USES_MBEDTLS_PSA` and `PSA_CRYPTO_PROVIDER_TFM` at the same time.
I even cannot imagine to what it might cause.

Mesh KConfig options should be reworked to use `PSA_CRYPTO_PROVIDER` correctly according to designed pattern.
Or shouldn't be changed at all.

This semicorrect variant is not the option.